packages/elements-core/src/components/MosaicTableOfContents/utils.ts

```diff
@@ -70,25 +70,10 @@ export function isGroup(item: TableOfContentsItem): item is TableOfContentsGroup
   return Object.keys(item).length === 2 && 'title' in item && 'items' in item;
 }
 export function isNodeGroup(item: TableOfContentsItem): item is TableOfContentsNodeGroup {
-  return (
-    Object.keys(item).length === 6 &&
```

Comment:
What is the reason for removal of this check?

Reply:
Optional version means that we can have 6 or 7 items

Reply:
Let's make sure we update the `TableOfContents` in Platform, because the current version checks for the length of this item. Currently ToC breaks in v2